mux/src/tab.rs

```diff
@@ -1861,6 +1945,42 @@ impl TabInner {
         None
     }
 
+    fn get_float_size(&self) -> TerminalSize {
```

Comment:
To me, `get_float_size` implies getting the size of the current float, but this is computing the desired size, so I think it should be renamed to make it more clear

```suggestion
    fn compute_float_size(&self) -> TerminalSize {
```